portals/admin/src/main/webapp/source/src/app/components/Workflow/WFRejectionPopup.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect, useState } from 'react';
+import TextField from '@mui/material/TextField';
+import FormDialogBase from 'AppComponents/AdminPages/Addons/FormDialogBase';
+import PropTypes from 'prop-types';
+import ClearIcon from '@mui/icons-material/Clear';
+import { FormattedMessage } from 'react-intl';
+
+/**
+ * Render a pop-up dialog to get the reason for WF rejection
+ * @param {JSON} props .
+ * @returns {JSX}.
+ */
+function WFRejectionPopup(props) {
+    const { referenceId, updateStatus } = props;
+    const [reason, setReason] = useState('');
+    const onChange = (e) => {
+        setReason(e.target.value);
+    };
+
+    useEffect(() => {
+        setReason('');
+    }, []);
+
+    const formSaveCallback = () => {
+        if (typeof updateStatus !== 'function') {
```

Comment:
Is it required to do this check here? `if (typeof updateStatus !== 'function') {`